### ISSUE TITLE
7b model on Colab: CUDA error 2 at /go/src/github.com/jmorganca/ollama/llm/llama.cpp/gguf/ggml-cuda.cu:8001: out of memory

### ISSUE THREAD

**Issue by nnWhisperer** [jmorganca/ollama#1492]
Hello,
On a Google Colab 50GB ram 16GB Vram T4 instance (problem persisted in V100 instance), I install ollama as follows:
```
!sudo curl -L https://ollama.ai/download/ollama-linux-amd64 -o /usr/bin/ollama
!sudo chmod +x /usr/bin/ollama
!ollama serve
```
On the terminal I say:
`ollama run yarn-mistral:7b-128k`
Log gives the following error while of the 16gbVRAM only 4.3 Gb of the VRAM was used :
`CUDA error 2 at /go/src/github.com/jmorganca/ollama/llm/llama.cpp/gguf/ggml-cuda.cu:8001: out of memory`

Following are the logs received:
[output.log](https://github.com/jmorganca/ollama/files/13654181/output.log)

**Comment by phalexo** [comment]
@BruceMacD 

The OOM bug is hiding somewhere in this folder ./ollama/llm/llama.cpp/gguf

When I copied over this folder from the tag v0.1.11 to the tag clone v0.1.12 the problem in v0.1.12 goes away.

**Comment by phalexo** [comment]
Try -DLLAMA_CUDA_FORCE_MMQ:BOOL=ON

**Comment by jmorganca** [comment]
Hi @nnWhisperer this should be fixed as of version 0.1.20. Note, filling large context windows will still cause potential OOM errors, this is being worked on in https://github.com/jmorganca/ollama/issues/1952
